dll/win32/shell32/lang/bg-BG.rc

```diff
@@ -497,14 +497,14 @@ IDD_LOG_OFF_FANCY DIALOGEX 0, 0, 208, 122
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_SHELLFONT | WS_BORDER | WS_POPUP
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CONTROL IDB_DLG_BG, IDC_STATIC, "STATIC", SS_BITMAP | SS_REALSIZECONTROL, 0, 0, 208, 122
-    CONTROL IDB_REACTOS_FLAG, IDC_STATIC, "STATIC", SS_BITMAP, 176, 1, 32, 26
+    CONTROL 0, IDC_DLG_BG, "STATIC", SS_OWNERDRAW, 0, 0, 208, 122
```

Comment:
Please use `CONTROL "",` (followed by the rest) instead of `CONTROL 0,`, here and everywhere else.